antora/package.json

```diff
@@ -22,7 +22,9 @@
   "scripts": {
     "build:base": "antora generate antora-playbook.yml",
     "build": "npm run build:base -- --clean --fetch",
-    "build:fast": "npm run build:base"
+    "build:fast": "npm run build:base",
+    "dev:links": "npm --prefix ../../ec-policies/antora/ec-policies-antora-extension link && npm --prefix ../../ec-cli/reference-antora-extension link && npm --prefix ../../ec-cli/rego-antora-extension link && npm --prefix ../../ec-cli/tekton-task-antora-extension link",
```

Comment:
`--prefix` here may not be working as expected. If I cd into that directory and run `npm link` then things seem to be working as expected.

Reply:
I noticed that links for some of the dependencies pointed to the global npm install directory, switched to using `cd` instead of `--prefix` and that went away, but introduced another issue, if `npm ci` wasn't invoked in an extension that has a dependency the transitive dependency would not be installed, so I added an additional `npm ci`.